Plater.lua

```diff
@@ -11301,4 +11311,4 @@ function Plater.OpenColorFrame()
 	a:SetWidth (totalWidth)
 end
 
---functiona enda
```

Comment:
Why the change to EOF (i assume)? Please revert.

Reply:
Editing via the website force adds eol